Source/Android/app/src/main/res/values/arrays.xml

```diff
@@ -643,28 +643,28 @@
     </integer-array>
 
     <string-array name="convertCompressionLevelZstdEntries">
-        <item>1</item>
-        <item>2</item>
-        <item>3</item>
-        <item>4</item>
-        <item>5</item>
-        <item>6</item>
-        <item>7</item>
-        <item>8</item>
-        <item>9</item>
-        <item>10</item>
-        <item>11</item>
-        <item>12</item>
-        <item>13</item>
-        <item>14</item>
-        <item>15</item>
-        <item>16</item>
-        <item>17</item>
-        <item>18</item>
-        <item>19</item>
-        <item>20</item>
-        <item>21</item>
-        <item>22</item>
+        <item>@string/compression_level_1</item>
+        <item>@string/compression_level_2</item>
+        <item>@string/compression_level_3</item>
+        <item>@string/compression_level_4</item>
+        <item>@string/compression_level_5</item>
+        <item>@string/compression_level_6</item>
+        <item>@string/compression_level_7</item>
+        <item>@string/compression_level_8</item>
+        <item>@string/compression_level_9</item>
+        <item>@string/compression_level_10</item>
+        <item>@string/compression_level_11</item>
+        <item>@string/compression_level_12</item>
+        <item>@string/compression_level_13</item>
+        <item>@string/compression_level_14</item>
+        <item>@string/compression_level_15</item>
+        <item>@string/compression_level_16</item>
+        <item>@string/compression_level_17</item>
+        <item>@string/compression_level_18</item>
+        <item>@string/compression_level_19</item>
+        <item>@string/compression_level_20</item>
+        <item>@string/compression_level_21</item>
+        <item>@string/compression_level_22</item>
```

Comment:
This is a little silly and should probably be redone at some point. But let's keep that out of this PR.